psdk_wrapper/src/modules/flight_control.cpp

```diff
@@ -903,10 +903,236 @@ void
 FlightControlModule::flight_control_generic_cb(
     const sensor_msgs::msg::Joy::SharedPtr msg)
 {
-  /** @todo implemnent generic control functionality */
+  /** @author Umesh Mane
+   * Contact: umeshmane280@gmail.com
+   */
+
```

Comment:
Generic comments on this method. I think it is very long and makes it a bit hard to read. My suggestion would be to split it in several different methods (e.g. one method to set the horizontal axis, one for vertical, another for yaw control).